src/store/modules/app.ts

```diff
@@ -25,7 +25,9 @@ const state: AppState = {
   slippage: lsGet(LS_KEYS.App.TradeSlippage, '0.01'),
   tradeLiquidity: lsGet(LS_KEYS.App.TradeLiquidity, LiquiditySelection.Best),
   transactionDeadline: lsGet(LS_KEYS.App.TradeDeadline, 20), // minutes
-  tradeInterface: lsGet(LS_KEYS.App.TradeInterface, TradeInterface.BALANCER)
+  // Force Balancer interface for now
+  tradeInterface:
+    TradeInterface.BALANCER /* lsGet(LS_KEYS.App.TradeInterface, TradeInterface.BALANCER), */
```

Comment:
This makes sure that if anyone has Gnosis interface saved - will bring him back to Balancer interface.